### PR TITLE
Allow for ambiata-us endpoint.

### PR DESCRIPTION
**Pull request by markhibberd**
(and corresponding us-east-1 aws region).

@nhibberd @olorin

### CODE REVIEW

**Comment by markhibberd** [comment]
Just adding some tests for render/parse things now. Getting this up to start getting things building.

**Comment by olorin** [comment]
LGTM, :+1: on :green_apple:.

File: test/Test/Ambiata/Cli/Data.hs [hunk 1]
```diff
@@ -34,6 +38,35 @@ prop_working_processing :: IncomingDir -> Bool
...
+prop_aws_region_explicit_us =
+  toAWSRegion AmbiataUs === NorthVirginia
+
+prop_endpoint_explicit_au :: Property
+prop_endpoint_explicit_au =
+  defaultAmbiataAPIEndpointFor AmbiataAu === AmbiataAPIEndpoint "https://api.ambiata.com"
+
+prop_endpoint_explicit_us :: Property
+prop_endpoint_explicit_us =
+  defaultAmbiataAPIEndpointFor AmbiataUs === AmbiataAPIEndpoint "https://us-api.ambiata.com"
```
Comment (olorin) [inline test/Test/Ambiata/Cli/Data.hs]: Need Eq [here](https://github.com/ambiata/ambiata-cli/blob/fcbd916b25561cfefa7cc65fc405a8c850088ac6/src/Ambiata/Cli/Data/Api.hs#L27) for these tests.

Comment (markhibberd) [inline test/Test/Ambiata/Cli/Data.hs]: Thanks (still sandboxing.....) .... .... ....

**Comment by nhibberd** [comment]
:facepunch:

**Pull request merged**